readthedocs/doc_builder/environments.py

```diff
@@ -951,22 +975,8 @@ def get_container_host_config(self):
         The object returned is passed to Docker function
         ``client.create_container``.
         """
-        binds = {
-            self.project.doc_path: {
-                'bind': self.project.doc_path,
-                'mode': 'rw',
-            },
-        }
-
-        if settings.GLOBAL_PIP_CACHE and settings.DEBUG:
-            binds.update({
-                self.project.pip_cache_path: {
-                    'bind': self.project.pip_cache_path,
-                    'mode': 'rw',
-                },
-            })
```

Comment:
We shouldn't be deleting code in this PR, unless it relates to getting this working I think?

Reply:
Ups. This was a mistake.